### PR TITLE
Add support for Python Six 1.9.0 library

### PR DESCRIPTION
**Pull request by snowcrshd**
This library is only available (officially) as a Mercurial repository.

Cloning it with Mercurial resulted in a very awkward python-six.yaml file,
so I opted to download the tarball directly from PyPI (as it is a reliable source).

PS: Maybe we should consider adding support to Mercurial (hg), when cloning from source repositories in the builds scripts.This situation is bound to repeat itself more and more often as we add support for new software.

### CODE REVIEW

**Comment by maurorodrigues** [comment]
Can one of the admins verify this patch?

**Comment by maurorodrigues** [comment]
you're totally right about it. It's already an issue at builds: open-power-host-os/builds#36

File: python-six/centOS/7.2/python-six.spec [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+%if 0%{?fedora} > 12 || 0%{?rhel} > 7
+%global with_python3 1
+
+%global __python3 python3
+%endif
+
+Name:           python-six
+Version:        1.9.0
+Release:        1%{?dist}
```
Comment (maurorodrigues) [inline python-six/centOS/7.2/python-six.spec]: release is smaller than the one provided by centOS.. can you check if you can upgrade it when centos python-sixis already installed?

**Comment by fabianorosas** [comment]
@snowcrshd @maurorodrigues Looking at the package list we compiled earlier, I think we're OK to use the CentOS version of this package.

File: python-six/centOS/7.2/python-six.spec (hunk 1 above)
Comment (snowcrshd) [inline python-six/centOS/7.2/python-six.spec]: I thought CentOS release was the same http://cbs.centos.org/koji/packageinfo?packageID=175

I'll check it out

**Comment by snowcrshd** [comment]
@fabianorosas @maurorodrigues I agree with Fabiano. We could simply use CentOS package.

Would it still need to be included in the Host OS versions repository?
For some reason the list we compiled earlier pointed it should be included in our repository.

**Comment by snowcrshd** [comment]
Well, there is no point in keeping this PR open.

I'll close it and we will use the CentOS version, since it is the same.

**Pull request closed without merging**